analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

```diff
@@ -188,7 +211,21 @@ genomic_assays_matched <- genomic_assays %>%
   filter(!descriptors %in% c("Diagnosis", "Progressive", "Recurrence", 
                              "Second Malignancy", "Deceased", "Not Reported",
                              "Unavailable", "Metastatic", "Residual", "")) %>%
-  mutate(experimental_strategy = gsub("RNA-Seq;","", experimental_strategy)) %>%
+ mutate(experimental_strategy = gsub("RNA-Seq;","", experimental_strategy),
+         timepoints_models = case_when(descriptors == "Diagnosis, Recurrence" ~ "Dx-Rec",
+                                       descriptors == "Diagnosis, Progressive" ~ "Dx-Pro",
+                                       descriptors == "Deceased, Diagnosis" ~ "Dx-Dec",
+                                       descriptors == "Deceased, Progressive" ~ "Pro-Dec",
+                                       descriptors == "Recurrence, Second Malignancy" ~ "Rec-SM",
+                                       descriptors == "Progressive, Recurrence" ~ "Pro-Rec",
+                                       descriptors == "Diagnosis, Second Malignancy" ~ "Dx-SM",
+                                       descriptors == "Deceased, Progressive, Recurrence" ~ "Pro-Rec-Dec",
+                                       descriptors == "Diagnosis, Progressive, Recurrence" ~ "Dx-Pro-Rec",
+                                       descriptors == "Deceased, Recurrence" ~ "Rec-Dec",
+                                       descriptors == "Deceased, Diagnosis, Progressive, Recurrence" ~ "Dx-Pro-Rec-Dec",
+                                       descriptors == "Deceased, Diagnosis, Recurrence" ~ "Dx-Rec-Dec",
+                                       descriptors == "Deceased, Diagnosis, Progressive" ~ "Dx-Pro-Dec",
+                                       descriptors == "Diagnosis, Unavailable" ~ "Dx-Unv")) %>% # to generate timepoints_models
```

Comment:
Some terms do not have match in descriptors. Especially terms start with `Deceased`, for example, I only spot `Diagnosis, Decreased` but not `Deceased, Diagnosis` in line 217. Also some short terms do not match the descriptors, such as line 222. I am not sure if it is your design or just some typo.

Reply:
When I wrote this code, I wasn't leveling the df by `tumor_descriptor`. I decided to do this later to make it easier to read the labels (since they would be ordered alphabetically). 

`timepoints <- c("Diagnosis", "Progressive", "Recurrence", "Deceased", "Second Malignancy", "Unavailable")
`

That means that cases that were previously identified as `Deceased, Diagnosis`, now they are as `Diagnosis, Deceased`. Hence, the code failed. Sorry for this omission. 

It is now fixed, see 690cd785ca31cd119dfd758b5bd3d493e4c8a0b7, e5615112e6858b79d81586431371c0289847bfb4 and 9ec274c423f65cc16a1ed86027ff9ad61b4fcfb1